crates/ruff_linter/src/fix/edits.rs

```diff
@@ -305,7 +305,7 @@ pub(crate) fn fits_or_shrinks(
     node: AnyNodeRef,
     locator: &Locator,
     line_length: LineLength,
-    tab_size: TabSize,
+    tab_size: IndentWidth,
```

Comment:
We could rename all these fields but here the `indent-width` is used as `tab-width` only, so keeping it as is felt correct.